### DISCUSSION TITLE
articles/2024/svelte5-safari12/

### DISCUSSION THREAD

**Discussion by giscus** [loopwerk/loopwerk.io#21] [Article discussions]
# articles/2024/svelte5-safari12/

Quite recently I upgraded a Svelte 4 project to Svelte 5, and soon afterwards I found some problems inside of Safari 12 and 13 that needed a tricky workaround.

https://www.loopwerk.io/articles/2024/svelte5-safari12/

**Comment by DjakaTechnology** [comment]
You saved me 6 hours of work, thanks!
